### PR TITLE
Send asset chooser

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x]  Don't hide device status bar
- [x] Cancel button is missing
- [x]  "Lumens" label shouldn't stay fixed on scroll
- [x] These label boxes should show a blue5 background
- [x] Rows should be more spaced out (56px high)

### CODE REVIEW

File: shared/common-adapters/box.native.js [hunk 1]
```diff
@@ -64,8 +64,8 @@ class Box2 extends React.Component<Box2Props> {
     return box2(this.props)
   }
 }
-const VBoxGap = ({gap}) => <View style={{height: globalMargins[gap]}} />
-const HBoxGap = ({gap}) => <View style={{width: globalMargins[gap]}} />
+const VBoxGap = ({gap}) => <View style={{flexShrink: 0, height: globalMargins[gap]}} />
```
Comment (chrisnojima) [inline shared/common-adapters/box.native.js]: don't allow these gaps to shrink. honestly i think its a better default for box2 in general but that should happen after the release (ticketed)

File: shared/common-adapters/section-list.desktop.js [hunk 2]
```diff
@@ -74,7 +74,10 @@ class SectionList extends React.Component<Props, State> {
     }
     const indexWithinSection = section.data.indexOf(item.item)
     return item.type === 'header' ? (
-      <Box key={item.key || key} style={this.props.stickySectionHeadersEnabled && styles.stickySectionHeader}>
+      <Box
+        key={item.key || key}
+        style={this.props.stickySectionHeadersEnabled ? styles.stickySectionHeader : null}
```
Comment (chrisnojima) [inline shared/common-adapters/section-list.desktop.js]: pass this through

File: shared/wallets/common/back-button.js [hunk 3]
```diff
@@ -5,38 +5,40 @@ import * as Styles from '../../styles'
 
 type Props = {
   onBack: () => void,
+  isOnWhiteBackground?: boolean,
```
Comment (chrisnojima) [inline shared/wallets/common/back-button.js]: this is used by the header which can be white sometimes

File: shared/wallets/send-form/choose-asset/index.js [hunk 4]
```diff
@@ -162,19 +146,20 @@ class ChooseAsset extends React.Component<Props, State> {
...
           <Header onBack={this.props.onBack} whiteBackground={true} />
-          <Box2 direction="vertical" fullWidth={true} style={styles.listContainer}>
-            <SectionList
+          <Kb.Box2 direction="vertical" fullWidth={true} style={styles.listContainer}>
+            <Kb.SectionList
               sections={sections}
               renderItem={this._renderItem}
               renderSectionHeader={this._renderSectionHeader}
               contentContainerStyle={styles.sectionList}
+              stickySectionHeadersEnabled={false}
```
Comment (chrisnojima) [inline shared/wallets/send-form/choose-asset/index.js]: no sticky sections

File: shared/wallets/send-form/choose-asset/index.js [hunk 5]
```diff
@@ -204,31 +189,30 @@ type DisplayChoiceProps = {
...
+    hoverColor={Styles.globalColors.blue4}
+    onClick={props.onClick}
+    style={styles.displayChoice}
+  >
+    <Kb.Box2 direction="horizontal" style={styles.choiceContainer} fullWidth={true}>
+      <Kb.Text
+        type="Body"
+        style={Styles.collapseStyles([styles.choice, props.selected ? styles.blue : undefined])}
+      >
+        {props.symbol === 'XLM' ? 'Purely strictly ' : 'Lumens displayed as '}
```
Comment (chrisnojima) [inline shared/wallets/send-form/choose-asset/index.js]: remove `(XLM)` which matches design and reduces the width which is too long for the 5s

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready
cc: @cecileboucheron

**Comment by cecileboucheron** [comment]
Looking good, a few remaining things:
* Cancel link should be blue, and there's a tiny space between the header and the section divider
image
* The line-height here seems busted, there shouldn't be any extra spacing between the 2 lines. Instead there should be an 8px spacing above and below the 2 lines.
image
* Checkmark should stand to the right of the row
image

**Comment by chrisnojima** [comment]
@cecileboucheron updated

**Comment by chrisnojima** [comment]
it shows for me so its already working

**Pull request merged**